packages/terminal/src/browser/terminal-preferences.ts

```diff
@@ -0,0 +1,54 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
```

Comment:
You might want to check the copyright.

Reply:
Ah yes!, Thanks.  Edited it to reflect my company. Cheers!